clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientBuilder.java

```diff
@@ -52,7 +53,6 @@ ZeebeClientBuilder applyEnvironmentVariableOverrides(
    *     to. Must be in format <code>host:port</code>. The default value is <code>0.0.0.0:26500
    *     </code> .
    */
-  @Deprecated
```

Comment:
🔧 Removing those depreciations will remove some depreciation warnings in code, for example if I move to use the Camunda-named equivalent class that extends this class here. I'd keep this depreciation therefore. Please correct me if I'm wrong.